schemas/com.shopify/update-product/jsonschema/1-0-0

```diff
@@ -0,0 +1,272 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "com.shopify",
+    "name" : "update-product",
+    "version" : "1-0-0",
+    "format" : "jsonschema"
+  },
+  "type" : "object",
+  "properties" : {
+    "body_html" : {
```

Comment:
If I understand correctly what it means - I tend to remove it entirely via adapter.